deployment/components/topology-updater/topologyupdater-mounts.yaml

```diff
@@ -4,9 +4,9 @@
   - name: host-sys
     hostPath:
       path: "/sys"
-  - name: kubelet-podresources-conf
+  - name: kubelet-state-files
```

Comment:
We want to add these to Helm chart as well

Reply:
Thanks!